setup.py

```diff
@@ -43,8 +43,8 @@ def finalize_options(self):
             sys.exit(1)
         install.finalize_options(self)
 
-version = "1.5.0"
-dependencies = ["cffi>=1.1.0; python_implementation != 'PyPy'"]
+version = "1.5.1"
```

Comment:
This happens automatically on release, no need to do it manually here.

Reply:
Ah, yes, I will revert the scm change which disabled it (needed a way to test locally).  I also would like this to work on pypy (which it now does) which is why I removed that.

Reply:
Oh, is this actually part of the cffi spec? My bad.

Reply:
You should be able to just `pip install .` to test locally; that's typically what I do.

Reply:
fixed.  I'd originally not realized that for long chains of dependencies you need to do --no-build-isolaton or it will go download the vanilla package from pypy during build.  But with that I can leave the version numbers alone and have it update automatically (will fix relevant version refs to xcffib in downstream projects with dependencies on xcffib)